dd-java-agent/instrumentation/undertow-1.4/src/main/java/datadog/trace/instrumentation/undertow/HandlerInstrumentation.java

```diff
@@ -0,0 +1,92 @@
+package datadog.trace.instrumentation.undertow;
+
+import com.google.auto.service.AutoService;
+import datadog.trace.agent.tooling.Instrumenter;
+import datadog.trace.bootstrap.instrumentation.api.AgentScope;
+import datadog.trace.bootstrap.instrumentation.api.AgentSpan;
+import io.undertow.server.HttpServerExchange;
+import net.bytebuddy.asm.Advice;
+import net.bytebuddy.description.type.TypeDescription;
+import net.bytebuddy.matcher.ElementMatcher;
+
+import static datadog.trace.agent.tooling.bytebuddy.matcher.DDElementMatchers.implementsInterface;
+import static datadog.trace.agent.tooling.bytebuddy.matcher.NameMatchers.named;
+import static datadog.trace.bootstrap.instrumentation.api.AgentTracer.activateSpan;
+import static datadog.trace.instrumentation.undertow.UndertowDecorator.DD_SPAN_ATTRIBUTE;
+import static datadog.trace.instrumentation.undertow.UndertowDecorator.DECORATE;
+import static net.bytebuddy.matcher.ElementMatchers.isMethod;
+import static net.bytebuddy.matcher.ElementMatchers.isPublic;
+import static net.bytebuddy.matcher.ElementMatchers.takesArgument;
+
+@AutoService(Instrumenter.class)
+public final class HandlerInstrumentation extends Instrumenter.Tracing {
+
+  public HandlerInstrumentation() {
+    super("undertow", "wildfly");
```

Comment:
```suggestion
    super("undertow", "undertow-1.4");
```

If you change the instrumentation name to `undertow-2.0` make this the same

Reply:
Fixed!